src/function.py

```diff
@@ -144,12 +160,16 @@ def _format_error(e, text):
                 raise BadRequestException(
                     _format_error(e, "Review the region endpoint")
                 )
-            elif e.status == 429:
-                print("There was a {} error. Reason: {}".format(e.status, e.message))
+            elif e.status in RETRYABLE_ERROR_CODES:
+                logger.error(f"There was a {e.status} error. Reason: {e.message}")
```

Comment:
Should a retry be logged as an error?

Reply:
You are right, maybe a warning would be more appropriate? And then only display an `error` when all retries have been attempted, what do you think?